### PR TITLE
Update docs for TiDB Dashboard

### PR DESCRIPTION
**Pull request by breeswish**
Signed-off-by: Wish <breezewish@outlook.com>

### What is changed, added or deleted? (Required)

- Fix: `**提示：**` does not have any effect, so it is removed.
- Enhance: Move the `慢查询页面显示 unknown field 错误` section down, since it contains lazy load images and causes anchors to be not effective when sections below are referenced

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/top-sql.md [hunk 1]
```diff
@@ -59,15 +59,13 @@ Top SQL 开启后会对集群性能产生轻微的影响（平均 3% 以内）
...
->
-> 除了通过图形化界面以外，你也可以配置 TiDB 系统变量 [`tidb_enable_top_sql`](/system-variables.md#tidb_enable_top_sql-从-v540-版本开始引入) 来启用 Top SQL 功能：
->
-> ```sql
-> SET GLOBAL tidb_enable_top_sql = 1;
-> ```
+除了通过图形化界面以外，你也可以配置 TiDB 系统变量 [`tidb_enable_top_sql`](/system-variables.md#tidb_enable_top_sql-从-v540-版本开始引入) 来启用 Top SQL 功能：
+
+```sql
+SET GLOBAL tidb_enable_top_sql = 1;
```
Comment (shichun-0415) [inline dashboard/top-sql.md:65]:
```suggestion
{{< copyable "sql" >}}

```sql
SET GLOBAL tidb_enable_top_sql = 1;
```

File: dashboard/top-sql.md [hunk 2]
```diff
@@ -110,13 +108,11 @@ Top SQL 开启后会对集群性能产生轻微的影响（平均 3% 以内）
...
->
-> 除了通过图形化界面以外，你也可以配置 TiDB 系统变量 [`tidb_enable_top_sql`](/system-variables.md#tidb_enable_top_sql-从-v540-版本开始引入) 来停用 Top SQL 功能：
->
-> ```sql
-> SET GLOBAL tidb_enable_top_sql = 0;
-> ```
+除了通过图形化界面以外，你也可以配置 TiDB 系统变量 [`tidb_enable_top_sql`](/system-variables.md#tidb_enable_top_sql-从-v540-版本开始引入) 来停用 Top SQL 功能：
+
+```sql
+SET GLOBAL tidb_enable_top_sql = 0;
```
Comment (shichun-0415) [inline dashboard/top-sql.md:114]:
```suggestion
{{< copyable "sql" >}}

```sql
SET GLOBAL tidb_enable_top_sql = 0;
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM. Thanks for your work.

File: dashboard/dashboard-faq.md [hunk 3]
```diff
@@ -145,3 +125,23 @@ NgMonitoring 是 TiDB v5.4.0 及以上集群中内置的高级监控组件，用
 请参见 TiDB Operator 文档中[启用持续性能分析](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/access-dashboard#启用持续性能分析)的步骤部署 NgMonitoring 组件。
 
 </details>
+
+### 慢查询页面显示 `unknown field` 错误
+
+集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。如果你的集群版本低于 v5.0.3 或 v4.0.14，可以根据以下步骤进行处理：
```
Comment (shichun-0415) [inline dashboard/dashboard-faq.md:131]:
```suggestion
集群升级后，如果慢查询页面出现 `unknown field` 错误，是由于升级后新版本 TiDB Dashboard 字段与浏览器缓存内的用户偏好设置的字段不兼容导致的。该问题已修复。如果你的集群版本低于 v5.0.3 或 v4.0.14，需要执行以下步骤清理浏览器缓存：
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**